flood_adapt/api/events.py

```diff
@@ -3,13 +3,41 @@
 from typing import Any
 
 from flood_adapt.dbs_controller import IDatabase
+from flood_adapt.object_model.hazard.event.synthetic import Synthetic
+from flood_adapt.object_model.interface.events import IEvent, ISynthetic
 
 
 def get_events(database: IDatabase) -> dict[str, Any]:
     # use PyQt table / sorting and filtering either with PyQt table or in the API
     return database.get_events()
 
 
+def get_event(name: str, database: IDatabase) -> IEvent:
+    return database.get_event(name)
+
+
+def create_synthetic_event(attrs: dict[str, Any], database: IDatabase) -> ISynthetic:
+    return Synthetic.load_dict(attrs)
+
+
+def save_synthetic_event(event: ISynthetic, database: IDatabase) -> None:
+    database.save_synthetic_event(event)
+
+
+def edit_synthetic_event(event: ISynthetic, database: IDatabase) -> None:
+    database.edit_synthetic_event(event)
```

Comment:
I was comparing this against Panos measures code. He does not have a save and edit function for each measure subclass. Can these also be generic save and edit functions for events?